packages/esm-patient-chart-app/src/clinical-views/utils/index.ts

```diff
@@ -3,7 +3,7 @@ import type { Encounter, ColumnDefinition, ConfigConcepts, EncounterTileColumn,
 import dayjs from 'dayjs';
 
 const calculateDateDifferenceInDate = (givenDate: string): string => {
-  return `${dayjs().diff(dayjs(givenDate), 'days')} days`;
+  return `${Math.abs(dayjs().diff(dayjs(givenDate), 'days'))} days`;
```

Comment:
While we have a helper for `age()` that wouldn't quite work here, I don't love this hard-coding of `days`. However, I think we can at least leverage [Intl.DurationFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DurationFormat) like:

```ts
return new Intl.DurationFormat(getLocale()).format({ days: Math.abs(dayjs().diff(dayjs(givenDate), 'days')) });
```

That should get us it formatted and localized correctly.

Reply:
@ibacher I get a type error saying `DurationFormat` does not exist, do we need to update something?

Reply:
Oh... I see. Well, ok. What happens if we call `age()` here instead?